.clinerules/03-cline-behavior.md

```
@@ -0,0 +1,42 @@
+
+- **一度に変更するファイルは最大3つまで**とする。それ以上になる場合は分割して提案する
+- 既存コードを削除・大幅変更する場合は、変更前に必ず理由を説明する
+- リファクタリングと機能追加は**同時に行わない**。別タスクとして分ける
+- テストコードがある場合は、実装コードと同時に更新する
+
+## 提案・回答のフォーマット
+
+コードを提示するときは以下の形式を使うこと:
+
+```
+【配置先】Assets/Scripts/Domain/Model/PlayerHp.cs
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)